lerobot/scripts/control_robot.py

```diff
@@ -442,12 +485,14 @@ def on_press(key):
 
         timestamp = time.perf_counter() - start_warmup_t
 
-    # Save images using threads to reach high fps (30 and more)
-    # Using `with` to exist smoothly if an execption is raised.
-    futures = []
+    # Save images using a worker process with writer threads to reach high fps (30 and more)
     num_image_writers = num_image_writers_per_camera * len(robot.cameras)
     num_image_writers = max(num_image_writers, 1)
-    with concurrent.futures.ThreadPoolExecutor(max_workers=num_image_writers) as executor:
+    frame_queue = multiprocessing.Queue()
```

Comment:
Minor: I've read that it's better to use `multiprocessing.manager().Queue()` instead of `multiprocessing.Queue()`:
https://www.geeksforgeeks.org/python-multiprocessing-queue-vs-multiprocessing-manager-queue/#

Reply:
Added a comment ;)
I prefer to keep `multiprocessing.Queue()` for now as I am using only 1 subprocess.